plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -678,6 +703,28 @@ public static Schema schemaFromMetadata(List<ColumnMetadata> columns)
         return new Schema(icebergSchema.asStructType().fields());
     }
 
+    public static Schema schemaFromViewColumns(TypeManager typeManager, List<ConnectorViewDefinition.ViewColumn> columns)
+    {
+        List<NestedField> icebergColumns = new ArrayList<>();
+        AtomicInteger nextFieldId = new AtomicInteger(1);
+        for (ConnectorViewDefinition.ViewColumn column : columns) {
```

Comment:
See if you can static import `ViewColumn`

Reply:
We can! fixed.